app/helpers/application_helper.rb

```diff
@@ -29,4 +29,8 @@ def govuk_back_link_to(url = :back, body = "Back")
   def link_to_runner(runner_url, form_id)
     "#{runner_url}/form/#{form_id}"
   end
+
+  def form_is_live(form)
+    form.live_at.present? && form.live_at < Time.zone.now
+  end
```

Comment:
I don't think this is needed we already have this method on the model https://github.com/alphagov/forms-admin/blob/main/app/models/form.rb#L14. 

Perhaps for the time being we dont have to check the live_at is in the past. If you need we should have it to save future work then could you make the changes to the model method?

Reply:
You're right - will get rid of the helper and move the check to the model. I think it's worth keeping the time check, easy to remove if we need to but also seems like a sensible bit of future-proofing / guard against weird stuff happening with our data.